Date: Fri, 5 Apr 2024 23:33:08 +0200
Subject: x86/microcode/AMD: Remove unused PATCH_MAX_SIZE macro

commit 3287c22957b401903e4933a81eea9191788da33b upstream.

Orphaned after

  05e91e721138 ("x86/microcode/AMD: Rip out static buffers")

No functional changes.

Signed-off-by: Borislav Petkov (AMD) <bp@alien8.de>
[ 6.1: Remove macro from arch/x86/include/asm/microcode_amd.h not
       arch/x86/kernel/cpu/microcode/amd.c since missing commit
       d02a0efd0f47 ("x86/microcode: Move core specific defines to local
       header")  in 6.1. ]
Signed-off-by: Suraj Jitindar Singh <surajjs@amazon.com>

diff --git a/arch/x86/include/asm/microcode_amd.h b/arch/x86/include/asm/microcode_amd.h
--- a/arch/x86/include/asm/microcode_amd.h
+++ b/arch/x86/include/asm/microcode_amd.h
@@ -41,8 +41,6 @@ struct microcode_amd {
 	unsigned int			mpb[];
 };
 
-#define PATCH_MAX_SIZE (3 * PAGE_SIZE)
-
 #ifdef CONFIG_MICROCODE_AMD
 extern void load_ucode_amd_early(unsigned int cpuid_1_eax);
 extern int __init save_microcode_in_initrd_amd(unsigned int family);
